channels/ie.m3u

```diff
@@ -25,15 +25,5 @@ https://d1h8rkqumz239n.cloudfront.net/oireachtas/ngrp:oireachtas.stream_all/play
 https://d33zah5htxvoxb.cloudfront.net/el/live/oirtv/hls_1280x720.m3u8
 #EXTINF:-1 tvg-id="RTENews.ie" tvg-country="IE" tvg-language="English" tvg-logo="https://i.imgur.com/OisW3m0.png" group-title="News",RTE News (1080p) [Offline]
 https://live.rte.ie/live/a/channel3/news.isml/.m3u8
-#EXTINF:-1 tvg-id="RTENews.ie" tvg-country="IE" tvg-language="English" tvg-logo="https://i.imgur.com/OisW3m0.png" group-title="News",RTE News [Offline]
-https://live.rte.ie/live/a/channel3/news.isml/news-audio_128k=128000-video=4499968.m3u8
```

Comment:
They've took an interesting turn.

Token https://www.rte.ie/servicelayer/api/anonymouslogin?isocode=IE
Prod search https://feed.entertainment.tv.theplatform.eu/f/1uC-gC/rte-prd-prd-all-listings/?byId=227353128156&fields=rtelisting$mediaPid
Get stream with token and media PID from Prod search result : https://link.eu.theplatform.com/s/1uC-gC/media/*insert pid*/?auth=*insert enormous token*&formats=mpeg-dash&format=smil&tracking=true